src/client/lazy-app/Compress/index.tsx

```diff
@@ -374,6 +376,14 @@ export default class Compress extends Component<Props, State> {
   }
 
   componentDidUpdate(prevProps: Props, prevState: State): void {
+    const wasLoading = prevState.loading;
+    const isLoading = this.state.loading || this.state.sides[0].loading || this.state.sides[1].loading;
+    const changedLoadingState = wasLoading !== isLoading;
+    if (changedLoadingState) {
+      updateDocumentTitle(loadingIndicator + this.state.source?.file.name);
```

Comment:
Just because the loading state has changed doesn't mean we're now in the loading state. It could have changed from loading to not-loading.

Reply:
Ah ok that makes more sense, will fix.